requirements/requirements-diffusion.txt

```diff
@@ -0,0 +1,6 @@
+accelerate
+diffusers
```

Comment:
I am somewhat concerned we can't be free with diffusers version, especially for attention quantization.
Could be worth checking the PR where I add support for Flux and its attention